scripts/e2e-monolith.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -x
+
+# Monolith version to test against
+export BUILDVERSION=2022.09.0-begin-289-g7f5834bbae22
+
```

Comment:
Is there any way to pull down the latest monolith version? Future enhancement?

Reply:
Or is this supposed to a fixed to specifically test against this version?

Reply:
Great question. I have a task to use API to select latest/stable version automatically: OKTA-528459 part of Epic OKTA-528454
We would get that in place before we turned this on as the default E2E script